carp.protocols.core/src/commonMain/kotlin/dk/cachet/carp/protocols/domain/sampling/DataTypeSamplingScheme.kt

```diff
@@ -19,6 +20,11 @@ abstract class DataTypeSamplingScheme<TSamplingConfigurationBuilder : SamplingCo
      */
     protected abstract fun createSamplingConfigurationBuilder(): TSamplingConfigurationBuilder
 
+    /**
+     * Get the [DataTypeMeasure]
+     */
+    fun measure() = DataTypeMeasure( type )
```

Comment:
This should be abstract, and will return `Measure`, not `DataTypeMeasure`. Concrete sampling schemes will return their own matching measures.

Reply:
Sure. But for now it will be removed though